.gitignore

```diff
@@ -1,5 +1,5 @@
 .DS_Store
 *.log
-dist
-lib/**.js
+lib
```

Comment:
We're just ignoring compiled file with `**.js`, I just want to keep the `lib` folder to avoid a potential error when running `gulp build`.